build_tools/cmake/build_and_test_asan.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
```

Comment:
I'm fine with the copy-paste for now, but it would be nice to common up some of these scripts. Maybe we should switch to Python and then we can have actually usable command line arguments. (no action necessary now)

Reply:
Yeah, I would like some fusion on this too. Separating the build and testing scripts with a few options to enable asan/tsan could fuse quite a bit.